### DISCUSSION TITLE
Any examples for v2?

### DISCUSSION THREAD

**Discussion by Confucian-e** [wevm/wagmi#3394] [Q&A]
It is hard to understand how to use wagmi v2. Are there some examples just like v1?

**Comment by tmm** [comment]
There's a connect wallet example: https://wagmi.sh/react/guides/connect-wallet#build-your-own

We'll add the [others](https://1.x.wagmi.sh/examples/send-transaction) back soon, but there aren't many changes.

**Comment by Nilbristi** [comment]
@tmm Can you provide an example of @wagmi/core too (Angular version)?
